app/webpack/users/edit/components/settings_item.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+const SettingsItem = ( {
+  header,
+  required,
+  children,
+  htmlFor
+} ) => (
+  <div className="row settings-item">
+    <div className="col-xs-12">
+      {( header && htmlFor ) && (
+        <label htmlFor={htmlFor}>
+          {header}
+          {required && " *"}
```

Comment:
This is fine, but you could make a more RTL-friendly version by assigning a `required` class to the `<label>` element and then doing something in css like
```css
label.required:after {
  content: "*";
  margin-left: 5px;
}
```

While the margin-left is still RTL-unfriendly, it makes it easier to support RTL later with something like

```css
label.required:after {
  content: "*";
  margin-left: 5px;
}
.rtl label.required:after {
  margin-left: 0px;
  margin-right: 5px;
}
```

Reply:
Updated as follows to incorporate the red asterisk in the Zeplin designs & avoid making the label deeppink. 

```
label.required {
  color: #333;
  &:after {
    content: "*";
    color: #ca5c53;
    margin-left: 5px;
  }
}
```

Reply:
You might want to use [$alert-red](https://github.com/inaturalist/inaturalist/blob/main/app/assets/stylesheets/colors.scss#L12) here.

Reply:
Done (and swapped in `$inat-green`)